apps/public-docsite-v9/src/Utilities/Theme/createCSSRuleFromTheme/createCSSRuleFromThemeBestPractices.md

```diff
@@ -1,7 +1,4 @@
-<details>
-<summary>
- Best Practices
-</summary>
+## Best Practices
```

Comment:
```suggestion
## Best practices
```

Reply:
good catch, I forgot to check outside `react-components` 😅